grafana/resource_oncall_integration.go

```diff
@@ -98,14 +98,59 @@ func ResourceOnCallIntegration() *schema.Resource {
 								Schema: map[string]*schema.Schema{
 									"channel_id": {
 										Type:        schema.TypeString,
-										Required:    true,
+										Optional:    true,
 										Description: "Slack channel id. Alerts will be directed to this channel in Slack.",
 									},
+									"enabled": {
+										Type:        schema.TypeBool,
+										Required:    true,
```

Comment:
Can we have it default to false. As it is, it's a breaking change because user configurations will now throw an error

Reply:
Makes sense. I set it to "true" by default because this is more expected behavior for users.